modules/ROOT/pages/_partials/gs-accessing-prometheus-grafana-proc.adoc

```diff
@@ -7,15 +7,17 @@ endif::[]
 = Accessing Prometheus and Grafana
 
 link:https://prometheus.io/[Prometheus] is an open-source systems and service monitoring and alerting toolkit that you can use to monitor services deployed in your Red Hat OpenShift environment. Prometheus collects and stores metrics from configured services at given intervals, evaluates rule expressions, displays the results, and can trigger alerts if a specified condition becomes true.
-Grafana is open source visualization and analytics software. It allows you to query, visualize, alert on, and explore your metrics.
+
+link:https://grafana.com/[Grafana] is open source visualization and analytics software. It allows you to query, visualize, alert on, and explore your metrics.
 
 .Prerequisite
-* You are logged in to the OpenShift console as the `customer-admin` user.
+* You are logged in to the {webapp} as the `customer-admin` user.
 
 .Procedure
-. In the *My Projects* section, click *View All*.
+. On the *All Services* tab, click the *Open console* button for *Red Hat OpenShift*.
+. Click the *My Projects* link.
 . Click the *Managed Service Monitoring* project.
-. From the sidebar, select *Applications* -> *Routes*.
-. To analyze and visualize your platform metrics using Grafana, click the hostname for _grafana-route_.
-+
-To query your platform metrics using Prometheus, click the hostname for _prometheus-route_.
+. In the sidebar, select *Applications* -> *Routes*.
+. In the *Routes* pane, you can choose to query or visualize the cluster metrics by clicking the host-names for different routes: 
```

Comment:
Here and in the sub-steps, "host name" should be two words (no hyphen, according to IBM manual of style).